dependencyManagement/build.gradle.kts

```diff
@@ -17,6 +17,7 @@ val DEPENDENCY_BOMS = listOf(
   "com.google.protobuf:protobuf-bom:3.21.12",
   "com.linecorp.armeria:armeria-bom:1.21.0",
   "com.squareup.okhttp3:okhttp-bom:4.10.0",
+  "com.squareup.okio:okio-bom:3.3.0", // applies to transitive dependencies of okhttp
```

Comment:
I did this to get rid of complaint about https://nvd.nist.gov/vuln/detail/CVE-2022-24329